sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -2423,6 +2450,9 @@ SIREPO.app.directive('appHeaderRight', function(appDataService, authState, appSt
         controller: function($scope, stringsService) {
             $scope.authState = authState;
 
+            $scope.getSlackLink = function() {
```

Comment:
I don't think this is necessary. You should be able to refer to `$scope.authState.slackUri`. 

Also, let's not change the name, i.e. it should be slackUri everywhere.

Reply:
1e63e3e61d6b75f5c183715519625d2fdca71b42